connector_prestashop_catalog_manager/models/product_template/exporter.py

```diff
@@ -242,53 +244,25 @@ def _get_product_category(self, record):
                 {'id': binder.to_external(category, wrap=True)})
         return ext_categ_ids
 
-    def _get_template_feature(self, record):
-        template_feature = []
-        attribute_binder = self.binder_for(
-            'prestashop.product.combination.option')
-        option_binder = self.binder_for(
-            'prestashop.product.combination.option.value')
-        for line in record.attribute_line_ids:
-            feature_dict = {}
-            attribute_ext_id = attribute_binder.to_external(
-                line.attribute_id.id, wrap=True)
-            if not attribute_ext_id:
-                continue
-            feature_dict = {'id': attribute_ext_id, 'custom': ''}
-            values_ids = []
-            for value in line.value_ids:
-                value_ext_id = option_binder.to_external(value.id,
-                                                        wrap=True)
-                if not value_ext_id:
-                    continue
-                values_ids.append(value_ext_id)
-            res = {'id_feature_value': values_ids}
-            feature_dict.update(res)
-            template_feature.append(feature_dict)
-        return template_feature
-
-    def _get_product_links(self, record):
-        links = []
-        binder = self.binder_for('prestashop.product.template')
-        for link in record.product_template_link_ids:
-            ext_id = binder.to_external(link.linked_product_template_id.id, wrap=True)
-            if ext_id:
-                links.append({'id': ext_id})
-        return links
+    def _get_product_image(self, record):
+        ext_image_ids = []
+        binder = self.binder_for('prestashop.product.image')
+        for image in record.image_ids:
+            ext_image_ids.append(
+                {'id': binder.to_external(image, wrap=True)})
+        return ext_image_ids
 
     @changed_by(
-        'attribute_line_ids', 'categ_ids', 'categ_id', 'product_link_ids'
+        'attribute_line_ids', 'categ_ids', 'categ_id', 'image_ids',
     )
     @mapping
     def associations(self, record):
         return {
             'associations': {
                 'categories': {
                     'category_id': self._get_product_category(record)},
-                'product_features': {
-                    'product_feature': self._get_template_feature(record)},
-                'accessories': {
-                    'accessory': self._get_product_links(record)},
```

Comment:
Why do you remove the export of product links and attributes ?

Reply:
@bguillot ,

About the export of product links, it requires module `product_template_multi_link`, that's why we removed it.

I'm sorry, it wasn't the smartest way.

Anyway, I think this should be done in another module to not add more dependencies to catalog manager.

We have done so in OCA/connector-prestashop#123 and OCA/connector-prestashop#132.

What do you think?

cc @MPoch-PlanetaTIC

Reply:
About the export of product attributes, I think "apples and oranges were mixing", I try to explain:

These are combinations and their option values:

![image](https://user-images.githubusercontent.com/39995504/65897502-afd0f600-e3af-11e9-832a-652837ebb52b.png)

And these are features and their values:

![image](https://user-images.githubusercontent.com/39995504/65897553-cd9e5b00-e3af-11e9-95ef-3d492bc91658.png)

Removed code was trying to save option values into product features:

https://github.com/akretion/connector-prestashop/pull/8/files#diff-da8e6cee6ea3159817655a0fef709826L245

Or that is what it seems to me :dizzy_face: